src/components/Calculator.scss

```diff
@@ -0,0 +1,42 @@
+.App {
+  display: flex;
+  justify-content: center;
+  margin-top: 25%;
+  width: 100vw;
+}
+
+.calculator {
+  display: flex;
+  flex-direction: column;
+  max-width: 25%;
+}
+
+::placeholder {
+  color: #fff;
+  font-size: 20px;
+}
+
+.input-area {
+  input {
+    width: 98%;
+    height: 90%;
+    margin-bottom: 25px;
+    direction: rtl;
+    background-color: grey;
+  }
+}
+
+.btn {
+  width: 25%;
+  height: 45%;
+  border: 1px solid rgb(217, 217, 217);
+}

```

Comment:
- [ ] Kindly make sure the `=` button has the same `background-color` as the provided design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/199838035-0572095d-f540-4f7e-8b54-b0dcce190009.png)  |  ![image](https://user-images.githubusercontent.com/74814780/199837884-510c8742-2809-471e-859a-838c26ded2c1.png)